quiz/rules/rules.css

```diff
@@ -0,0 +1,36 @@
+@import "../../common.css";
+
+/* Main Section */
+.main {
+  margin: 2rem 15rem;
+  padding: 1rem;
+  border: 2px solid var(--color-black);
+  border-radius: 1rem;
+}
+
+.image-container {
+    width: 23rem;
+    height: 14rem;
+}
+
+.image-container img {
+    border-radius: 1rem;
+}
+
+.flex {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.width-15 {
+    width: 16rem;
+}
```

Comment:
the class name is 15 but the value is 16

Reply:
done